Content.Client/_DV/AACTablet/UI/AACBoundUserInterface.cs

```diff
@@ -0,0 +1,29 @@
+using Content.Shared._DV.AACTablet;
+using Content.Shared._DV.QuickPhrase;
+using Robust.Client.UserInterface;
+using Robust.Shared.Prototypes;
+
+namespace Content.Client._DV.AACTablet.UI;
+
+public sealed class AACBoundUserInterface : BoundUserInterface
+{
+    [ViewVariables]
+    private AACWindow? _window;
+
+    public AACBoundUserInterface(EntityUid owner, Enum uiKey) : base(owner, uiKey)
```

Comment:
a little bit redundant, can move the arguments to the constructor (i.e. `public sealed class AACBoundUserInterface(EntityUid owner, Enum uiKey))`